collector/lib/CollectorConfig.cpp

```diff
@@ -283,6 +287,8 @@ void CollectorConfig::InitCollectorConfig(CollectorArgs* args) {
   HandleAfterglowEnvVars();
   HandleConnectionStatsEnvVars();
   HandleSinspEnvVars();
+  std::filesystem::path configMapFilePath = runtime_configmap_path.valueOr("/run/collector_runtime_config/config.json");
```

Comment:
1. Why JSON and not YAML like other configurations in Kubernetes and StackRox?

2. nit: You can use refer to the config as `/run/collector/...` instead, this gives the directory more flexibility to be used for different collector configs. The file name could be more explicit and refer to the runtime config.

```suggestion
  std::filesystem::path configMapFilePath = runtime_configmap_path.valueOr("/run/collector/runtime_config.json");
```